scripts/superdesk-search/react.js

```diff
@@ -862,6 +901,11 @@
                             scope.$apply(function() {
                                 activityService.start(this.props.activity, {data: {item: this.props.item}});
                             }.bind(this));
+
+                            if (_.contains(['spike', 'unspike'], this.props.activity.when)) {
+                                $rootScope.$broadcast('multi:updateCount', this.props.item._id);
```

Comment:
imo this would be better in some `item:spike` and `item:unspike` event handler, so it works also for other users via websockets and not only current user